src/components/Navbar/SiteNavigation.tsx

```diff
@@ -79,6 +80,10 @@ export default function SiteNavigation() {
                         <NavLanguageSelector className="md:order-2" />
                         <ThemePicker className="md:order-3" />
                         <QuickProjectSearch className="md:order-1" />
+                        <TransactionsList
+                          className="relative md:order-4"
```

Comment:
I wonder if we can 'bake' these class names into the TransactionList itself. Do you think we'll need the flexibility to reuse the tx list with different classes elsewhere?

Reply:
good call, done